app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -105,12 +105,11 @@ class ThreadFragment : Fragment() {
     private val mainViewModel: MainViewModel by activityViewModels()
     private val threadViewModel: ThreadViewModel by viewModels()
 
+    private val twoPaneFragment inline get() = parentFragment as TwoPaneFragment
```

Comment:
This could be abstracted to lower coupling but maybe ThreadFragment is really really always meant to be displayed inside a TwoPaneFragment so yeah...

Reply:
ThreadFragment is really really always meant to be displayed inside a TwoPaneFragment :)